io.github._0xzer0x.qurancompanion.yml

```diff
@@ -0,0 +1,26 @@
+app-id: io.github._0xzer0x.qurancompanion
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: quran-companion
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
Why does it need access to the entire home?

Reply:
should i limit it using `--persist` ?

Reply:
it uses a hidden directory for downloading/configuration

Reply:
replaced it with the `--persist` option
https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access